src/app/services/itinerary.service.ts

```diff
@@ -0,0 +1,22 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { Itinerary } from '../models/itinerary';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ItineraryService {
+
+  constructor(private httpClient: HttpClient) { }
+
+  hostURL = 'http://localhost:3000/';
+
+  public lista(): Observable<Itinerary[]> {
+    var url=this.hostURL + 'itineraries'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ispp-yourney/angular-app/pullRequest?prid=7122565)
```suggestion
    var url=this.hostURL + "itineraries"
```